### ISSUE TITLE
Can't mount FAT32 partition from AmigaOS

### ISSUE THREAD

**Issue by TuKo1982** [PiStorm/hdf2emu68#2]
For some reason, the FAT32 partition created by hdf2emu68 can't be mounted in AmigaOS (w/ FAT95). Reformatting that single partition from Windows fixes it.

A little thing makes it somehow different.

**Comment by TomCat666** [comment]
Yes, just tried. It seems the FatFS system generates a bit different filesystem which cannot be mounted on amiga :( The MBR is the same, the actual content of the filesystem is not

**Comment by TomCat666** [comment]
Paraj found a fix, it is commited on my fork of the util. Claude will pull and merge to main branch when he has time.
Mounts with fat95 properly on real amiga now
